### PR TITLE
Addition of purpose of UG, amendment of introduction and quick start

### PR DESCRIPTION
**Pull request by Johnson-Yee**
#55 
Also, reformatted enter button.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -32,33 +31,50 @@
 - [6. Glossary](#6-glossary)
 
 ## 1. What is Nav@NUS?
+Are you new to NUS? <br>
+Are you searching for ways to get around NUS all squeezed up in front of a tiny information board?<br>
+We have just the right solution for you!<br><br>
+Introducing **Nav@NUS**, you new navigation assistant!
```
Comment (wamikamalik) [inline docs/UserGuide.md]:
```suggestion
Introducing **Nav@NUS**, your new navigation assistant!
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
-| |\ \  | |    / /\ \ \ \     / /
-| | \ \ | |   / /__\ \ \ \   / /
-| |  \ \| |  / /----\ \ \ \ / /
-|_|   \ \_| / /      \ \ \_V_/   @NUS 
-```
+![Logo](UG_Figures/Nav@NUSLogo.jpg)
+## Opening words
+Welcome to Nav@NUS application's user guide! <br><br>
+The purpose of this user guide is to provide you with all the necessary information to use this application to navigate
+around NUS's Kent Ridge Campus via the school's shuttle service.<br>
```
Comment (wamikamalik) [inline docs/UserGuide.md:6]: I think the bus routes have BTC bus stops oso?

Comment (Johnson-Yee) [reply docs/UserGuide.md:6]: Dine options also include BTC campus?

Comment (wamikamalik) [reply docs/UserGuide.md:6]: Yah

**Comment by wamikamalik** [comment]
Might have to pull from master 1st. I have updated /descfav. Not reflected here

File: docs/UserGuide.md (hunk 2 above)
Comment (Johnson-Yee) [reply docs/UserGuide.md:6]: Alright will change all mentions of Kent Ridge Campus to NUS Campus

**Pull request closed without merging**